### ISSUE TITLE
sudo geonode-updateip fails on fresh Ubuntu 16.04 install

### ISSUE THREAD

**Issue by jamieodonnell** [GeoNode/geonode#3874]
I have followed through the [Quick Installation Guide](http://docs.geonode.org/en/master/tutorials/install_and_admin/quick_install.html) and am at the point at which I have to set the correct IP address for Geonode to deploy on. 

`sudo geonode-updateip localhost ` gives me the following response: 

```
$ sudo geonode-updateip localhost
Replacing SITEURL value with 'localhost' in /etc/geonode/local_settings.py ... done.
Adding entry for 'localhost' in /usr/share/geoserver/data/printing/config.yaml ... 'localhost' already found to the printing whitelist.
Setting up oauth
/usr/sbin/geonode-updateip: line 154: restart: command not found
Waiting Tomcat Service to Restart...
Waiting Apache HTTPD Service to Restart...
Inspecting the available layers in GeoServer ...
GET http://localhost:8080/geoserver/rest/workspaces.xml
GET http://localhost:8080/geoserver/rest/workspaces/geonode/datastores.xml
GET http://localhost:8080/geoserver/rest/workspaces/geonode/coveragestores.xml
GET http://localhost:8080/geoserver/rest/workspaces/geonode/wmsstores.xml
Found 0 layers, starting processing
Finished processing 0 layers in 0.0 seconds.
0 Created layers
0 Updated layers
0 Failed layers
0.000000 seconds per layer
```

I can access Geoserver with no issues though localhost returns a 400 error.

**Comment by jamieodonnell** [comment]
Does this change the procedures in the [quick install instructions?](http://docs.geonode.org/en/master/tutorials/install_and_admin/quick_install.html)

**Comment by frafra** [comment]
The patch fixes the problem, but it is not available in 2.8.x nor in the PPA. Looks like a backport is needed for 2.8.
